src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -49,49 +36,85 @@
-        c.command('list', 'list_service_tier_advisors')
-        c.command('show', 'get_service_tier_advisor')
+###############################################
+#                sql elastic-pool             #
+###############################################
 
 elasticpools_ops = create_service_adapter('azure.mgmt.sql.operations.elastic_pools_operations',
                                           'ElasticPoolsOperations')
 
 with ServiceGroup(__name__, get_sql_elasticpools_operations, elasticpools_ops) as s:
-    with s.group('sql elastic-pools') as c:
-        c.command('create', 'create_or_update')
+    with s.group('sql elastic-pool') as c:
+        c.custom_command('create', 'elastic_pool_create')
         c.command('delete', 'delete')
         c.command('show', 'get')
         c.command('list', 'list_by_server')
         c.generic_update_command('update', 'get', 'create_or_update')
 
-    with s.group('sql elastic-pools db') as c:
-        c.command('list', 'list_databases')
-        c.command('show', 'get_database')
-        c.command('show-activity', 'list_database_activity')
-
 recommanded_elastic_pools_ops = \
     create_service_adapter('azure.mgmt.sql.operations.recommended_elastic_pools_operations',
                            'RecommendedElasticPoolsOperations')
 
-with ServiceGroup(__name__, get_sql_recommended_elastic_pools_operations,
-                  recommanded_elastic_pools_ops) as s:
-    with s.group('sql elastic-pools recommended') as c:
-        c.command('show', 'get')
-        c.command('show-metrics', 'list_metrics')
-        c.command('list', 'list')
+#with ServiceGroup(__name__, get_sql_recommended_elastic_pools_operations,
+#                  recommanded_elastic_pools_ops) as s:
+    ## Recommended elastic pools will not be included in the first batch of GA commands
+    #with s.group('sql elastic-pool recommended') as c:
+    #    c.command('show', 'get')
+    #    c.command('show-metrics', 'list_metrics')
+    #    c.command('list', 'list')
+
+    #with s.group('sql elastic-pool recommended db') as c:
+    #    c.command('show', 'get_databases')
+    #    c.command('list', 'list_databases')
+
+###############################################
+#                sql server                   #
+###############################################
+
+server_operations = create_service_adapter('azure.mgmt.sql.operations.servers_operations',
+                                           'ServersOperations')
+
+with ServiceGroup(__name__, get_sql_servers_operation, server_operations) as s:
+    with s.group('sql server') as c:
+        c.command('create', 'create_or_update')
+        c.command('delete', 'delete')
+        c.command('show', 'get_by_resource_group')
+        ## Usages will not be included in the first batch of GA commands
+        #c.command('show-usage', 'list_usages')
+        c.command('list', 'list_by_resource_group')
+        c.generic_update_command('update', 'get_by_resource_group', 'create_or_update')
```

Comment:
admin_login and location can not be updated, should there be a separate command for update that omits these parameters?